src/OrchardCore.Modules/OrchardCore.Spatial/wwwroot/Scripts/coordinate-leaflet-field.js

```diff
@@ -0,0 +1,52 @@
+/*
+** NOTE: This file is generated by Gulp and should not be edited directly!
+** Any changes made directly to this file will be overwritten next time its asset group is processed by Gulp.
+*/
+
+function addMapPicker() {
+  var lat = $('[data-latitude]').val();
+
+  var _long = $('[data-longitude]').val();
+
+  var mapCenter = [40.866667, 34.566667];
+  var zoom = 0;
+
+  if (lat && _long) {
+    mapCenter = [lat, _long];
+    zoom = 14;
+  }
+
+  var map = L.map('map', {
+    center: mapCenter,
+    zoom: zoom
+  });
+  L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+    attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+  }).addTo(map);
+  var marker = L.marker(mapCenter, {
+    draggable: true
+  }).addTo(map);
+
+  function updateMarker(lat, lng) {
+    marker.setLatLng([lat, lng]).bindPopup("Location :  " + marker.getLatLng().toString()).openPopup();
+    return false;
+  }
+
+  ;
+  map.on('click', function (e) {
+    $('[data-latitude]').val(e.latlng.lat);
+    $('[data-longitude]').val(e.latlng.lng);
+    updateMarker(e.latlng.lat, e.latlng.lng);
+  });
+
+  var updateMarkerByInputs = function updateMarkerByInputs() {
+    return updateMarker($('[data-latitude]').val(), $('[data-longitude]').val());
+  };
+
+  $('[data-latitude]').on('input', updateMarkerByInputs);
+  $('[data-longitude]').on('input', updateMarkerByInputs);
+}
+
+$(document).ready(function () {
+  addMapPicker();
+});
```

Comment:
Please add eol.

Reply:
File is generated like this by the gulp pipeline and the original asset has a eol?